src/components/AddItem.js

```diff
@@ -1,5 +1,88 @@
+import { addDoc, collection } from 'firebase/firestore';
+import { useState, useEffect } from 'react';
+import { db } from '../lib/firebase.js';
+
 const AddItem = () => {
-  return <h1>Add Item</h1>;
+  const [itemName, setItemName] = useState('');
+  const [buyAgainTime, setBuyAgainTime] = useState(null);
+  const [successMessage, setSuccessMessage] = useState('');
+
+  const itemToAdd = {
+    name: itemName,
+    'last purchased': null,
+    'next purchase': Number(buyAgainTime),
+    token: 1234,
```

Comment:
I believe this is where we will want to integrate getting the token from local storage. I pulled this branch and replaced line 14 with this line on my machine:

token: localStorage.getItem('token'),

I was able to then add a new item (celery) to list and the token that was on my machine (sage window alpha) was written to the database.

Reply:
Thanks Chris!